peer.go

```diff
@@ -1,516 +1,518 @@
-// Copyright (c) 2013-2014 The btcsuite developers
-// Use of this source code is governed by an ISC
-// license that can be found in the LICENSE file.
-
 package main
 
 import (
-	"bytes"
-	"container/list"
 	"fmt"
-	"io"
-	prand "math/rand"
-	"net"
-	"strconv"
-	"sync"
-	"sync/atomic"
 	"time"
 
 	"github.com/btcsuite/btcd/addrmgr"
-	"github.com/btcsuite/btcd/blockchain"
 	"github.com/btcsuite/btcd/database"
+	"github.com/btcsuite/btcd/peer"
 	"github.com/btcsuite/btcd/wire"
 	"github.com/btcsuite/btcutil"
 	"github.com/btcsuite/btcutil/bloom"
-	"github.com/btcsuite/go-socks/socks"
-	"github.com/davecgh/go-spew/spew"
-)
-
-const (
-	// maxProtocolVersion is the max protocol version the peer supports.
-	maxProtocolVersion = 70002
```

Comment:
We should discuss this a bit.  I suspect we'll need both the max peer version in the peer package and a max server protocol version in btcd proper.  The reason is that with them being separate it is entirely possible the peer supports a higher version number than the server.